### PR TITLE
Update azug.be landing page with updated agenda

### PR DESCRIPTION
**Pull request by tomkerkhove**
Update azug.be landing page with updated agenda.

Do not merge before 7/05 in evening

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,10 +14,7 @@ excerpt: "The Belgium Azure User Group focuses on knowledge sharing and networki
 
 ## Upcoming events
 
-**2019-05-06 - //BUILD Live stream** - Join with your peers and watch the //BUILD 2019 live stream together. [Register here!](https://www.azug.be/events/2019/05/06/build-live-stream){: .btn .btn--success}
-{: .notice--info}
-
-**2019-05-07 - Binding your serverless clients & 6 years of Bootstrapping a SaaS on Azure** - Want to learn how you can write your own Azure Function bindings from Jan de Vries? Or hear Maarten Balliauw talk about 6 years of Bootstrapping a SaaS on Azure? Then this event is for you! [Register here!](https://www.azug.be/events/2019/05/07/binding-your-serverless-clients){: .btn .btn--success}
+**2019-12-13 & 14 - CloudBrew 2019** - Two days of Azure, beers and networking! [Register here!](https://www.cloudbrew.be/){: .btn .btn--success}
```
Comment (tomkerkhove) [inline index.md]: Should we add a page to our events section on this as well? If so, I can take this up.

Comment (maartenba) [reply index.md]: Note registrations are not open yet :-) Maybe "more info"?

And yes, page in events section could be nice (can be copied from last year, mostly)

**Comment by tomkerkhove** [comment]
Fair enough!

File: index.md (hunk 1 above)
Comment (tomkerkhove) [reply index.md]: Will do in seperate PR!

**Comment by tomkerkhove** [comment]
Actually, I think we can merge it already since registrations are already closed.

**Pull request merged**